File: Data Structures & Algorithms/concatenation-of-array/submission-0.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int[] ans = new int[nums.length * 2];

        for (int i = 0, j = nums.length; j < ans.length; i++, j++) {
            ans[i] = nums[i];
            ans[j] = nums[i];
        }

        return ans;
    }
}

File: Data Structures & Algorithms/three-integer-sum/submission-0.py
class Solution:
    def threeSum(self, nums: List[int]) -> List[List[int]]:
        nums.sort()
        print(nums)     

        result = []
        for i, n in enumerate(nums):
            if i > 0 and nums[i] == nums[i - 1]:
                continue

            target = -n
            j = i + 1
            k = len(nums) - 1
            while j < k:
                twoSum = nums[j] + nums[k]
                if twoSum == target:
                    result.append([nums[i], nums[j], nums[k]])
                    j += 1
                    k -= 1

                    while j < k and nums[j] == nums[j - 1]:
                        j += 1
                    while j < k and nums[k] == nums[k + 1]:
                        k -= 1
                elif twoSum < target:
                    j += 1
                elif twoSum > target:
                    k -= 1


        return result